Add docs and api-docs info for gcp replication locations sync, github environments, github organizations beta, and add sys/activation-flags page (#28463)
hashicorp/vault: website/content/docs/sync/vercelproject.mdx

</Note>

<Note>

> Vercel Project environment variables only support single-value secrets, and Vault syncs secrets differently
> depending on whether you have configured `secret-key` or `secret-path` [granularity](/vault/docs/sync#granularity):
> 
> - `secret-key` granularity splits KVv2 secrets from Vault into key-value pairs
>   and stores the pairs as distinct entries in Vercel. For example,
>   `secrets.key1="val1"` and `secrets.key2="val2"`.
> 
> - `secret-path` granularity stores secrets as a single JSON string that contains
>   all of the associated key-value pairs. For example, `{"key1":"val1", "key2":"val2"}`.
> 
> For Vercel Project, the granularity defaults to `secret-key`.

Reviewer: ```suggestion
Vault syncs secrets differently depending on whether you have configured 
`secret-key` or `secret-path` [granularity](/vault/docs/sync#granularity):

- `secret-key` granularity splits KVv2 secrets from Vault into key-value pairs
  and stores the pairs as distinct entries in Vercel. For example,
  `secrets.key1="val1"` and `secrets.key2="val2"`.

- `secret-path` granularity stores secrets as a single JSON string that contains
  all the associated key-value pairs. For example, `{"key1":"val1", "key2":"val2"}`.

Since Vercel projects limit environment variables to single-value secrets, the
sync granularity defaults to `secret-key` for Vercel projects.
```

I don't think this needs to be wrapped in a `Note` aside. Especially since there's a note directly before it
